application/common/models/Method.php

```diff
@@ -61,4 +93,156 @@ public function getRawPhoneNumber()
     {
         return preg_replace('/,/', '', $this->value);
     }
+
+    /**
+     * @param integer $userId
+     * @param string $type
+     * @param string $value
+     * @return Method
+     * @throws \Exception
+     */
+    public static function createAndSendVerification($userId, $type, $value)
+    {
+        $log = [
+            'class' => __CLASS__,
+            'method' => 'createAndSendVerification',
+            'user_id' => $userId,
+            'type' => $type,
+        ];
+
+        $method = new Method();
+        $method->user_id = $userId;
+        $method->type = $type;
+        $method->value = $value;
+
+        if ($type == self::TYPE_PHONE) {
+            $log['value'] = Utils::maskPhone($value);
+        } elseif ($type == self::TYPE_EMAIL) {
+            $log['value'] = Utils::maskEmail($value);
+        } else {
+            $log['value'] = 'invalid type';
```

Comment:
Were these just for logging an error, or did you forget to add code to log success?